stress/shell/src/main/java/alluxio/stress/cli/RegisterWorkerBench.java

```diff
@@ -0,0 +1,224 @@
+
+  private final InstancedConfiguration mConf = InstancedConfiguration.defaults();
+
+  private List<LocationBlockIdListEntry> mLocationBlockIdList;
+
+  private Deque<Long> mWorkerPool = new ArrayDeque<>();
+
+  @Override
+  public void prepare() throws Exception {
+    // The task ID is different for local and cluster executions
+    // So including that in the log can help associate the log to the run
+    LOG.info("Task ID is {}", mBaseParameters.mId);
+
+    // Generate block IDs heuristically
+    Map<BlockStoreLocation, List<Long>> blockMap =
+        RpcBenchPreparationUtils.generateBlockIdOnTiers(mParameters.mTiers);
+    BlockMasterClient client =
+            new BlockMasterClient(MasterClientContext
+                    .newBuilder(ClientContext.create(mConf))
+                    .build());
+    mLocationBlockIdList = client.convertBlockListMapToProto(blockMap);
+
+    // Prepare these block IDs concurrently
+    LOG.info("Preparing blocks at the master");
+    RpcBenchPreparationUtils.prepareBlocksInMaster(blockMap, getPool(), mParameters.mConcurrency);
+    LOG.info("Created all blocks at the master");
+
+    // Prepare worker IDs
+    int numWorkers = mParameters.mConcurrency;
+    mWorkerPool = RpcBenchPreparationUtils.prepareWorkerIds(client, numWorkers);
+    Preconditions.checkState(mWorkerPool.size() == numWorkers,
+        "Expecting %s workers but registered %s",
+        numWorkers, mWorkerPool.size());
+    LOG.info("Prepared worker IDs: {}", mWorkerPool);
+  }
+
+  /**
+   * @param args command-line arguments
+   */
+  public static void main(String[] args) {
+    mainInternal(args, new RegisterWorkerBench());
+  }
+
+  private RpcTaskResult simulateRegisterWorker(alluxio.worker.block.BlockMasterClient client) {
+    RpcTaskResult result = new RpcTaskResult();
+    long i = 0;
+
+    if (mWorkerPool == null) {
+      result.addError("Worker ID pool is null");
+      return result;
+    }
+    if (mWorkerPool.isEmpty()) {
+      result.addError("No more worker IDs for use");
+      return result;
+    }
+    long workerId = mWorkerPool.poll();
+
+    // Each client will simulate only one worker register RPC
+    // Because the number of concurrent register RPCs is the variable we want to control
+    // And we want these RPCs to invoke at roughly the same time
+    runOnce(client, result, i, workerId);
+
+    return result;
+  }
+
+  private List<String> simulateTierAliases(String tierConfig) {
+    String[] parts = tierConfig.split(";");
+    LOG.info("Simulate {} tiers with config {}", parts.length, tierConfig);
+    List<String> aliases = new ArrayList<>();
+    for (int i = 0; i < parts.length; i++) {
+      aliases.add(TIER_ALIASES.get(i));
+    }
+    return aliases;
+  }
+
+  private Map<String, Long> simulateCapacityMap(List<String> tierAliases) {
+    Map<String, Long> capMap = new HashMap<>();
+    for (String t : tierAliases) {
+      capMap.put(t, CAPACITY);
+    }
```

Comment:
More concise and readable using Guava Maps:

`Maps.toMap(tierAliases, (tier) -> CAPACITY)`